example.gulp-config-local.js

```diff
@@ -0,0 +1,11 @@
+// This is a config file for your local Gulp settings which
+// should not been committed to the repo.
+// 1. Copy example.gulp-config-local.js to gulp-config-local.js.
+// 2. Set your local values.
+
+module.exports = {
+  localHost: function() {
+    // Set the URL of your local Drupal install. It is used by BrowserSync.
+    return 'http://umdrupal.loc';
```

Comment:
Should we make this localhost by default since that is the most default configuration possible?

Reply:
I do not think that this is the most default URL. It can be anything. I just wanted to give en example. Maybe I should update the comment above the return statement like this:
// Set**s** the URL…

Reply:
Isn't http://localhost the most default since it exists on most machines by default?

Reply:
I never had a (Drupal) site on my Mac using the URL http://localhost